packages/react/src/experimental/F0VersionHistory/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface VersionAuthor {
+  firstName: string
+  lastName: string
+  src?: string
+}
+
+export interface Version {
+  id: string
+  author: VersionAuthor
+  timestamp: Date
+  onClick?: () => void
+}
+
+export interface CurrentVersion {
```

Comment:
what is the difference between current version and other ones? I mean, could we model it with just one type?